Makefile

```diff
@@ -109,6 +109,9 @@ status_handler_test : status_handler_test.o status_handler.o request_handler.o s
 status_counter_test : status_counter_test.o status_counter.o gmock_main.a
 	$(CXX) $(CPPFLAGS) $(CXXFLAGS) -lpthread $^ -o $@ -lboost_system
 
+redirect_handler_test : redirect_handler_test.o redirect_handler.o request_handler.o gmock_main.a
+	$(CXX) $(CPPFLAGS) $(CXXFLAGS) -lpthread $^ -o $@ -lboost_system
+
 
 TESTS = config_parser_test server_test echo_handler_test \
```

Comment:
I notice that you guys forgot to add redirect_handler_test to the TESTS variable. Our make test will run all tests included in this variable.